### PR TITLE
Add api to allow deletion of items older than a provided timestamp

### PR DESCRIPTION
**Pull request by cliftonc**
To allow newsfeed cleanup, e.g. on a scheduled basis select users and remove feed items > 30 days old.

### CODE REVIEW

File: api/feed/index.js [hunk 1]
```diff
@@ -256,6 +256,22 @@ module.exports = function (api) {
...
   }
 
+  function removeFeedsOlderThan (keyspace, user, time, next) {
+    async.map(FEEDS, function (timeline, cb) {
+      _removeFeedsOlderThanFromTimeline(keyspace, timeline, user, time, cb);
+    }, next);
+  }
+
+  function _removeFeedsOlderThanFromTimeline (keyspace, timeline, user, time, next) {
+    var options = {raw: true, olderThan: client.generateTimeId(time), pageSize: 1000};
```
Comment (cliftonc) [inline api/feed/index.js]: Page size = 1000 ... because I didn't want to implement paging and this seemed like a reasonable size to find and delete.

**Comment by cliftonc** [comment]
@dmorgantini can you take a look?

**Comment by dmorgantini** [comment]
@cliftonc I'm not certain that this provides the value we are looking for.  Presuming that we want to be able to reduce the number of posts that we are storing, we need to be able to do this across millions of users rather than for an individual user.

**Comment by cliftonc** [comment]
Thats the next step - I'm taking this in small steps.

**Pull request merged**